core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -175,6 +178,31 @@ public void removeBlock(long blockId) {
     mToRemoveBlocks.remove(blockId);
   }
 
+  /**
+   * Adds a new worker lost storage paths.
+   *
+   * @param tierAlias the tier alias
+   * @param dirPath the lost storage path
+   */
+  public void addLostStorage(String tierAlias, String dirPath) {
+    List<String> paths = mLostStorage.getOrDefault(tierAlias, new ArrayList<>());
+    paths.add(dirPath);
+    mLostStorage.put(tierAlias, paths);
+  }
+
+  /**
+   * Adds new worker lost storage paths.
+   *
+   * @param lostStorage the lost storage to add
+   */
+  public void addLostStorage(Map<String, List<String>> lostStorage) {
+    for (Map.Entry<String, List<String>> entry : lostStorage.entrySet()) {
+      List<String> paths = mLostStorage.getOrDefault(entry.getKey(), new ArrayList<>());
```

Comment:
Here will reduce many map get and put operations because we will `paths.addAll(List<String>)` instead of one by one `paths.add(String storage` in the `addLostStorage(tierAlias, dirPath`